dagobah/core/dagobah.py

```diff
@@ -19,6 +19,7 @@ class Dagobah(object):
     instance, as well as top-level parameters such as the
     backend used for permanent storage.
     """
+    JIJ_DELIM = '%_|JIJ_DELIMITER|_%'
```

Comment:
Future-you might thank you for making this a tad less verbose when you're going through debug logs, presumably `%_|` or similar would be enough to prevent conflicts with any sane person's job naming?